api/src/main/java/org/eclipse/microprofile/config/spi/ConfigBuilder.java

```diff
@@ -115,6 +115,16 @@
      */
     <T> ConfigBuilder withConverter(Class<T> type, int priority, Converter<T> converter);
 
+    /**
+     * Set the config profile.
+     * Once the config profile is set, the config property {@code %profile.config.name} overrides the property {@code config.name} 
+     * in the same config source. 
+     * The config value for the property {@code config.name} is retrieved from a config source with the highest ordinal. 
+     * The value is set by the property {@code %profile.config.name} if exists. Otherwise, the value is taken from the property {@code config.name}.
+     * @param profile the config profile
+     * @return
+     */
+    ConfigBuilder withProfile(String profile);
     /**
```

Comment:
A small detail, but there should be a blank line in between the method names for consistency with the rest of the file.

Reply:
ok. fixed.